cmd/image-factory/cmd/service.go

```diff
@@ -178,6 +181,8 @@ func runMetricsServer(ctx context.Context, logger *zap.Logger, eg *errgroup.Grou
 }
 
 func buildArtifactsManager(ctx context.Context, logger *zap.Logger, opts Options) (*artifacts.Manager, error) {
+	var err error
+	var artifactsManager *artifacts.Manager
```

Comment:
not sure why this declaration is even needed

Reply:
Removed the declaration